src/service/action-impl.js

```diff
@@ -259,6 +263,18 @@ export class ActionService {
         this.addTargetPropertiesAsDetail_(deferredEvent);
         debouncedInput(deferredEvent);
       });
+    } else if (name == 'input-throttled') {
```

Comment:
awesome. let's also update documentation in [amp-actions-and-events.md](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md) with the new event.